ui/src/helpers/utils.js

```diff
@@ -0,0 +1,8 @@
+import intersection from 'lodash/intersection';
+
+export function arrayElementsComparetion(array1,array2){
```

Comment:
change to ES6 syntax :
export const arrayElementsComparetion = (array1,array2) => {